### ISSUE TITLE
VideoPress: Tab navigation gets hidden inside non-playing videos.

### ISSUE THREAD

**Issue by tommusrhodus** [Automattic/jetpack#32655]
### Impacted plugin

VideoPress

### Quick summary

Navigating through the page with TAB feels like it gets hijacked by the VideoPress iFrame, as visually, the only component available is the play button, however all other controls in the iFrame (which are non-visible in this state) also pick up focus.

This gives the impression that TAB navigation has become broken, due to focussing on hidden elements.

### Steps to reproduce

1. Start at https://jetpack.com/support/jetpack-videopress/changing-a-videos-thumbnail/
2. TAB navigate through the page until you get focus on the play icon on the video in this page.
3. Do not hit enter to trigger playing.
4. TAB once more, notice your TAB focus is no longer visible.

It will take another 9 TABS before your focus re-appears on the play button, and a further TAB to continue on the page.

### A clear and concise description of what you expected to happen.

TAB navigation should be limited to visually available elements. I expected to TAB onto the play button, and then TAB onto the next focus element on the page if I chose not to hit ENTER and play.

### What actually happened

TAB navigation disappeared onto focusable elements in the VideoPress iFrame that are visually hidden.

### Impact

All

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple, Atomic, Self-hosted
